### DISCUSSION TITLE
Can I solve the CVE ( about postgresql   and kafka) by this way ?

### DISCUSSION THREAD

**Discussion by DeBruyne2020** [apache/skywalking#12035] [Q&A]
I have a question for help,🙏

We are using Apache Skywalking 9.7 . This week was published the security issue [CVE-2024-1597](https://nvd.nist.gov/vuln/detail/CVE-2024-1597) (base score 10 - critical) in Postgre JDBC client and CVE-2023-25194 ( about kafka ),

In oap-libs of Apache Skywalking 9.7, the version of postgresql jar is 42.4.1,   the version of kafka-clients jar is 2.8.1.  Both cannot  fix the CVE(about  postgresql and kafka)  above. 

My question is :   Can I just remove the postgresql-42.4.1.jar and  kafka-clients-2.8.1.jar in oap-libs  to evade the the CVE(about  postgresql and kafka)?  (The datasource of our skywalking oap is es and we dont't use postgresql , and we don't use kafka in our application-system or skywalking).          Can skywalking work normal if I remove the postgresql-42.4.1.jar and  kafka-clients-2.8.1.jar in oap-libs  of skywalking ?( we don't use postgresql and we don't use kafka)

**Comment by wu-sheng** [comment]
Yes, if you are 100% sure we are not using Kafka and Postgre relative features. It is safe. OAP doesn't rely on that.
